spec/features/incoming_mail_handler_spec.rb

```diff
@@ -82,19 +82,9 @@
 
     resp = handler.handle(mandrill_event)
 
-    expect(resp.action).to eq(IncomingMail::Response::COMMENT)
-    expect(resp.comment.proposal.existing_observation_for(user)).to be_present
-    expect(resp.comment.proposal.existing_approval_for(user)).not_to be_present
-    expect(my_approval.user).to_not eq(my_proposal.individual_approvals.last.user)
-
-    comment_recipients = [
-      my_proposal.requester.email_address, # comment
-      my_proposal.individual_approvals.last.user.email_address, # comment
-      my_approval.user.email_address, # comment
-    ]
-
-    expect(deliveries.length).to eq(comment_recipients.size)
-    expect(deliveries.map{|m| m.to.first}.sort).to eq(comment_recipients.sort)
+    expect(resp.action).to eq(IncomingMail::Response::FORWARDED)
+    expect(deliveries.length).to eq(1)
+    expect(my_proposal.has_subscriber?(user)).to be_falsey
```

Comment:
what do you think about writing this expectation as `expect(my_proposal).not_to have_subscriber(user)` ? 

I aways find `falsey` a bit weird because I can't remember if it is the same as `eq false` or not and also spelling (both `falsey` and `falsy` work in RSpec which is bizarre IMO)...so I try to avoid it when I can